src/redux/reducers/imgDailyReducer.js

```diff
@@ -1,7 +1,7 @@
 import { FETCH_IMAGES_SUCCESS, IS_LOADING } from '../types'
 
 const initialState = {
-  image: {},
+  image: '',
```

Comment:
we need a constant for **''**, sometimes it's more clear:
like:
```
const EMPTY_STRING = ''

const initialState = {
  image: EMPTY_STRING
}
```

please add the **EMPTY_STRING** constant in **common/constants** directory